CHANGELOG.md

```diff
@@ -22,6 +22,7 @@ For older changes see the [archived Singularity change log](https://github.com/a
   include the libraries needed for maximum support of compression algorithms
   by squashfuse_ll.
 - Add automated tests for OpenSUSE Leap and Tumbleweed and Debian Bookworm.
+- Fixed typo in `nvliblists.conf` (`libnvoptix.so.1` -> `libnvoptix.so`)
```

Comment:
Unfortunately now I notice a typo in the CHANGELOG.  `nvliblists.conf` should be `nvliblist.conf`.  Not nearly as important as a typo in code, but let's fix it.

Reply:
Fixed in https://github.com/apptainer/apptainer/pull/2527/commits/a19fa01527a8914839b8d1649688f83c61ba9ad2